lib/core/utils/widgets/custom_toast.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flowery_e_commerce/core/styles/colors/my_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+
+class CustomToast {
+  static void showToast({
+    required String message,
+    required Color backgroundColor,
+    required ToastGravity gravity,
+    required IconData icon,
```

Comment:
Plese refactor this line to replace IconData with Widget to can use animated image in future like lotti files or image assets